config/routes.rb

```diff
@@ -312,11 +312,9 @@
       put '/doc_auth/:step' => 'doc_auth#update'
       get '/doc_auth/link_sent/poll' => 'doc_auth#doc_capture_poll'
       get '/capture_doc' => 'capture_doc#index'
+      get '/capture-doc' => 'capture_doc#index', as: :capture_doc_dashes
       get '/capture_doc/:step' => 'capture_doc#show', as: :capture_doc_step
       put '/capture_doc/:step' => 'capture_doc#update'
-      get '/capture-doc/:step' => 'capture_doc#show',
-          # sometimes underscores get messed up when linked to via SMS
```

Comment:
WDYT about keeping the comment for the new `/capture-doc` route?

Reply:
> WDYT about keeping the comment for the new `/capture-doc` route?

Restored the comment in the rebased b5f1334ea68533771810bbf9b83022da279d3482.